ui/src/app/routing/app-routing.tsx

```diff
@@ -287,11 +293,21 @@ export const AppRoutingComponent: React.FunctionComponent<RoutingProps> =
                     exact={false}
                     guards={[signInGuard, disabledGuard(isUserDisabledInDb)]}
                   >
-                    <SignedInPage intermediaryRoute={true} routeData={{}} />
+                    {!doesUserNeedToAcceptTOS && (
+                      <SignedInPage intermediaryRoute={true} routeData={{}} />
+                    )}
                   </AppRoute>
                 </Switch>
               </AppRouter>
             )}
+            {doesUserNeedToAcceptTOS && (
```

Comment:
This is probably me not understanding the flow, but when I comment this block out, and try to create an account, I see this:
![image](https://user-images.githubusercontent.com/24514630/159998846-fe70290b-ebb8-48c1-adfb-9b9d8563f153.png)
Isn't the component in the screenshot the one that is being referenced here?